infra/content/kXml.class.php

```diff
@@ -42,7 +42,13 @@ class kXml
 {
 	public static function getXslEnabledPhpFunctions()
 	{
-		return array('date', 'gmdate', 'strtotime','urlencode','xml_load_for_xslt');
+		return array('date', 'gmdate', 'strtotime','urlencode','xml_load_for_xslt', 'myXmlUtils::kGmdate', 'myXmlUtils::kDate');
+	}
+	
+	public static function transformSafePhpFunction()
+	{
+		$xslt = str_replace("'gmdate'", "'myXmlUtils::kGmdate'", $xslt);
```

Comment:
where do you get `$xslt` from? you don't get it as function param...

Reply:
Fixed.